### ISSUE TITLE
NA values in pbs data

### ISSUE THREAD

**Issue by kellijohnson-NOAA** [DFO-NOAA-Pacific/surveyjoin#32]
When running the following code to get data on Pacific Hake
```
surveyjoin::cache_data()
surveyjoin::load_sql_data()

cached_data <- surveyjoin::get_data(
  itis_id = 164792,
  regions = c("afsc", "nwfsc", "pbs")
)
```
I get 
```
> cached_data|>dplyr::filter(is.na(year)) 
# A tibble: 233 × 21
   event_id   itis catch_numbers catch_weight region scientific_name common_name
      <dbl>  <int>         <dbl>        <dbl> <chr>  <chr>           <chr>
 1  5618345 164792             0         0    pbs    merluccius pro… north paci…
 2  5618346 164792             0         0    pbs    merluccius pro… north paci…
 3  5618347 164792             0         0    pbs    merluccius pro… north paci…
 4  5618348 164792             0         0    pbs    merluccius pro… north paci…
 5  5618349 164792             0         0    pbs    merluccius pro… north paci…
 6  5618350 164792             1         1.72 pbs    merluccius pro… north paci…
 7  5618351 164792             0         0    pbs    merluccius pro… north paci…
 8  5618352 164792             0         0    pbs    merluccius pro… north paci…
 9  5618353 164792             0         0    pbs    merluccius pro… north paci…
10  5618354 164792             0         0    pbs    merluccius pro… north paci…
# ℹ 223 more rows
# ℹ 14 more variables: survey_name <chr>, date <chr>, pass <int>, vessel <chr>,
#   lat_start <dbl>, lon_start <dbl>, lat_end <dbl>, lon_end <dbl>,
#   depth_m <dbl>, effort <dbl>, effort_units <chr>, performance <chr>,
#   bottom_temp_c <dbl>, year <int>
# ℹ Use `print(n = ...)` to see more rows
```
Some of these records even have positive catches for hake. I am guessing they just are not aligning with the haul data properly but I am not sure.
This seems like something that could be written into a general test to make sure that the downloaded/cached data do not have NA values in at least some of the columns like year, survey_name, region, ...

**Comment by ericward-noaa** [comment]
I'm just checking this against the current version of the dataset. There's 8 records total (4 from AFSC, 4 from PBS that meet this criteria). The AFSC ones have negative event IDs -- and I'm guessing should be deleted

**Comment by Lewis-Barnett-NOAA** [comment]
Hmm, I can't verify completely because my IT-controlled browser settings aren't letting me authenticate to google drive via tidyverse right now.

I did notice that the data on the google drive was old so I updated that (also in data-raw).

There are 4 hauls from Alaska surveys with NA end latitude and longitude, and there is no catch associated, so those are the culprits I will update with those hauls removed.  The negative event_id is fine, there are more like that.
